docs/develop/topics/administration_panel.md

```diff
@@ -0,0 +1,432 @@
+
+```bash
+vim invenio-rdm-records/setup.cfg
+```
+
+```ini
+[options.entry_points]
+invenio_administration.views =
+    invenio_rdm_records_oai_list = invenio_rdm_records.administration.views.oai:OaiPmhListView
+    invenio_rdm_records_oai_edit = invenio_rdm_records.administration.views.oai:OaiPmhEditView
+    invenio_rdm_records_oai_create = invenio_rdm_records.administration.views.oai:OaiPmhCreateView
+    invenio_rdm_records_details = invenio_rdm_records.administration.views.oai:OaiPmhDetailView
+```
+
+### Create custom view
+
+A custom view can be created by inheriting directly from `AdminView`.
+#### Usage
+
+```python
+from invenio_administration.views.base import AdminView
+
+class MyCustomView(AdminView):
+    """My custom view."""
+
+    name = "customview"
+    category = "My category"
+    template = "invenio_administration/index.html"
+    url = None
+    menu_label = "Custom View"
+    icon = "user"
+```
+
+The class defined for the custom view must be registered as an entry point, as follows:
+
+```bash
+vim invenio-rdm-records/setup.cfg
+```
+
+```ini
+[options.entry_points]
+invenio_administration.views =
+    invenio_module_admin_custom_view = invenio_module.administration.views:MyCustomView
+```
+
+For the full attributes list and description visit [reference docs](../../reference/administration_reference.md)
+
+### Customisation: dashboard view
+
+The dashboard view can be configured by creating a view that inherits from the `AdminView` and updating the config variable `ADMINISTRATION_DASHBOARD_VIEW` with the new declared view.
+
+```python
+ADMINISTRATION_DASHBOARD_VIEW = (
+    "invenio_administration.views.dashboard.AdminDashboardView"
+)
+```
+
+If there is no need to provide any additional template context, dashboard view can be also overridden by adding new jinja template in your local instance, as follows: `templates/invenio_administration/index.html`. 
+
+### Customisation: jinja templates
+
+The views are defined by jinja templates, and this can be easily extendable and overridable by redefining their blocks.
+
+Depending on the desired view to overridden, the correct template should be extended:
+
+```html
+<!-- Details page -->
+{% extends "invenio_administration/details.html" %}
+<!-- Edit page -->
+{% extends "invenio_administration/edit.html" %}
+<!-- Search page -->
+{% extends "invenio_administration/search.html" %}
+<!-- Create page -->
+{% extends "invenio_administration/create.html" %}
+<!-- Dashboard page -->
+{% extends "invenio_administration/index.html" %}
+```
+
+In each page we can override or extend different blocks, mainly there are 2 blocks that can be overrided to display our custom content:
+
+- `admin_page_content`: Main content each page. It's present in all the views.
+- `javascript`: Main block to place any javascript content. It's important to always use {{ super() }} inside this block to inherit all the javascript needed to render the rest of the page.
+
```

Comment:
I saw that there is an example snippet below. It would be nice to mention that it can be seen below or add here and empty/simple snippet with the blocks.